src/LibHdb++.cpp

```diff
@@ -1,40 +1,44 @@
-//=============================================================================
-//
-// file :        LibHdbL.cpp
-//
-// description : Source for the LibHdb library.
-//
-// Author: Graziano Scalamera
-//
-//
-//=============================================================================
+/* Copright (C) 2014-2017
```

Comment:
Copright => Copyright in the whole project

Reply:
Ugh